Documentation/UploadBinaryData.md

```diff
@@ -11,24 +11,59 @@ A [solution to this problem] which has been adopted by ITK is to store binary
 files, such as images, in a separate location outside the Git repository, then
 download the files at build time with [CMake].
 
-A "content link" file contains an identifying [SHA512 hash]. The content
+A "content link" file contains an identifying [Content Identifier (CID)]. The content
 link is stored in the [Git] repository at the path where the file would exist,
-but with a `.sha512` extension appended to the file name. [CMake] will find
+but with a `.cid` extension appended to the file name. [CMake] will find
 these content link files at **build** time, download them from a list of server
 resources, and create symlinks or copies of the original files at the
 corresponding location in the **build tree**.
 
+The [Content Identifier (CID)] is a self-describing hash following the
+[multiformats] standard created by the
+Interplanetary Filesystem ([IPFS]) community. A file
+with a CID for its filename is content-verifiable. Locating files
+according to their CID makes content-addressed, as opposed to
+location-addressed, data exchange possible. This practice is the
+foundation of the decentralized web, also known as the dWeb or Web3. By
+adopting Web3, we gain:
+
+  - Permissionless data uploads
+  - Robust, redundant storage
+  - Local and peer-to-peer storage
+  - Scalability
+  - Sustainability
+
+Contributors to the examples upload their data through an a simple web app
+that utilizes an easy-to-use, permissionless, free service, [web3.storage].
+
+Data used in the examples Git repository is periodically tracked in a
```

Comment:
Only applies to the examples? Or does this apply to test data as well?

Reply:
Test data -- good catch! Fixed.